argocd-boostrap/bootstrap.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+echo "checking ArgoCD version on the server"
+argocd version --short --client  
+
+echo "change argocd-server service type to load balancer"
+export ARGO_SERVER=$(kubectl get svc -n argocd -l app.kubernetes.io/name=argocd-server -o name) 
+
+echo "set argocd password as env variable" 
+export ARGO_PASSWORD=$(kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d)
+
+echo "login to the server and change password"
+argocd login $ARGO_SERVER --username admin --password $ARGO_PASSWORD 
```

Comment:
Let's add running the job to the Argo AddOn and add actual bootstrapping. New pwd should be passed in. Github secret (if any) may be supplied by the user.

Reply:
added step to change password and added options for users that may require private repositories